apps/backend/src/modules/ai-assistant/ai-assistant.controller.ts

```diff
@@ -343,6 +345,21 @@ class AIAssistantController extends BaseController {
 			},
 		});
 
+		this.addRoute({
+			handler: (options) =>
+				this.acceptMultipleTasks(
+					options as APIHandlerOptions<{
+						body: AIAssistantCreateMultipleTasksDto;
+						user: UserDto;
+					}>,
+				),
+			method: "POST",
+			path: AIAssistantApiPath.CHAT_ACCEPT_MULTIPLE_TASKS,
```

Comment:
```suggestion
			path: AIAssistantApiPath.CHAT_ACCEPT_TASKS,
```
We will use this even to accept only 1 task